services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -103,6 +124,14 @@ async def postprocess(self):
         await self._process_task_log()
         log.debug("Post-Processing DONE")
 
+    async def cleanup(self):
+        log.debug("Cleaning...")
+        await self._post_messages(LogType.LOG, "[sidecar]Cleaning...")
```

Comment:
MINOR: ``self._post_message`` acts here as a logger, right? why not create a prefix and format everything in the imlementation? 
```python
await self._post_message(LogType.LOG, who="sidecar", msg="Cleaning")
# or
await self._post_message(LogType.LOG, "sidecar", "Cleaning")
# or simply bind it

await def sidecar_log(self, msg):
    log.debug(msg)
    await self._post_messages(LogType.LOG, who="sidecar", msg=msg)
# ...
await self.sidecar_log("Cleaning...")
```


Look what happens now
![image](https://user-images.githubusercontent.com/32402063/91350654-463bb080-e7e7-11ea-91e6-3fff1e3f6443.png)

Reply:
well yes.. but there I'm not re-doing the sidecar... :rofl: there is already another WIP for this